build.xml

```diff
@@ -843,6 +843,7 @@ TODO:
 
     <path id="quick.bin.tool.path">
       <path refid="quick.repl.build.path"/>
+      <path refid="quick.repl-jline.build.path"/>
```

Comment:
the quick.bin target copies all class files from classes/repl-jline  to classes/repl